ts/store/middlewares/analytics.ts

```diff
@@ -263,6 +264,22 @@ const trackAction =
         }
         break;
       }
+      case getType(migrateToPaginatedMessages.request): {
+        return mp.track("MESSAGES_MIGRATION_START", {
+          total: action.payload.total
```

Comment:
> **total**: numero dei messaggi da migrare

![Schermata 2022-03-30 alle 12 04 19](https://user-images.githubusercontent.com/822471/160806538-a445d26f-b4cb-40ce-a2ce-688913c5c5bf.png)
As far I know, it seems we are tracing `MessageStatus | undefined` and not a `number`
If it is true, maybe this is not we want

Reply:
Good catch. What we want is the size of that store. Fixed [here](https://github.com/pagopa/io-app/pull/3865/commits/af5257fed09d564aafb42740e28f7fd8009eaa56).